WebService/v1/Controllers/SimulationsController.cs

```diff
@@ -133,8 +163,20 @@ public async Task<SimulationApiModel> GetAsync(string id)
                 throw new BadRequestException("No data or invalid data provided");
             }
 
+            var patchServiceModel = patch.ToServiceModel(id);
+
+            if (patchServiceModel.Enabled == false)
+            {
+                patchServiceModel.Statistics = new Services.Models.SimulationStatistics
```

Comment:
I would avoid doing this:
* the Agent already stores the stats when stopping the simulation
* if there are multiple simulation running (in future) there are multiple `rateReporter` objects, one per simulation

Reply:
I tried but that does not work. When a simulation is manually stopped the Stop/Update methods in Agent are not triggered

Reply:
:white_check_mark: